components/org.wso2.carbon.identity.oauth2.grant.organizationswitch/src/main/java/org/wso2/carbon/identity/oauth2/grant/organizationswitch/OrganizationSwitchGrant.java

```diff
@@ -242,32 +238,26 @@ private Optional<org.wso2.carbon.user.core.common.User> getFederatedUserFromResi
         }
     }
 
-    /**
-     * Update the token binding reference with the new token id.
-     *
-     * @param tokenId Token id.
-     * @param tokenBindingRef Token binding reference.
-     * @throws IdentityOAuth2Exception If an error occurs while updating the token binding reference.
-     */
-    private void updateTokenBindingRef(String tokenId, String tokenBindingRef)
+    private void checkOrganizationIsAllowedToSwitch(String tokenIssuedOrgId, String tokenRequestedOrgId)
```

Comment:
change the variables names, current org id & switch org id. fix other places also.
add a method comment also.

Reply:
Fixed by - https://github.com/wso2-extensions/identity-oauth2-grant-organization-switch/pull/14/commits/c5813e8893b0c0f0e0cdad0d6903ebbcc53ec4a1